### PR TITLE
Web UI: Fix betanag popover position

### PR DESCRIPTION
**Pull request by M4rtinK**
First set the popover placement priority to prefer the bottom direction
and then make it narrower so it actually fits if shown under the betanag
bubble.

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Thank you, looks good to me.

**Comment by martinpitt** [comment]
I took a deeper look at these npm install failures, see https://github.com/cockpit-project/cockpituous/pull/498 . Retried. Please retry other PRs which got affected by that.

File: ui/webui/src/components/AnacondaHeader.jsx [hunk 1]
```diff
@@ -36,7 +36,8 @@ export const AnacondaHeader = ({ beta, title }) => {
         ? (
             <Popover
               headerContent={_("This is unstable, pre-release software")}
-              minWidth="40rem"
+              minWidth="30rem"
+              flipBehavior={["bottom", "right", "left", "top"]}
```
Comment (poncovka) [inline ui/webui/src/components/AnacondaHeader.jsx:40]: Have you tried to set `position={PopoverPosition.auto}` instead?

Comment (M4rtinK) [reply ui/webui/src/components/AnacondaHeader.jsx]: > Have you tried to set `position={PopoverPosition.auto}` instead?

Thanks, that works better! But still needs the `minWidth` to be set to 30rem or else it still won't fit.

**Review by VladimirSlavik — APPROVED** [review]

**Pull request merged**